src/stuf/stuf_zds/client.py

```diff
@@ -18,11 +18,7 @@
 from requests import RequestException, Response
 
 from openforms.config.models import GlobalConfiguration
-from openforms.logging.logevent import (
-    stuf_zds_failure_response,
-    stuf_zds_request,
-    stuf_zds_success_response,
-)
+from openforms.logging import logevent
```

Comment:
Next time, do these refactors in a separate PR - they clutter the diff for PR review.